docs/resources/waf_domain.md

```diff
@@ -71,7 +71,7 @@ The following arguments are supported:
 * `certificate_name` - (Optional, String) Specifies the certificate name. This parameter is mandatory
   when `client_protocol` is set to HTTPS.
 
-* `policy_id` - (Optional, String, ForceNew) Specifies the policy ID associated with the domain. If not specified, a new
+* `policy_id` - (Optional, String) Specifies the policy ID associated with the domain. If not specified, a new
   policy will be created automatically. Changing this create a new domain.
```

Comment:
```suggestion
  policy will be created automatically.
```
Add description `Changing this create a new domain.` Only when field is `ForceNew`.